pom.xml

```diff
@@ -972,14 +997,28 @@
       </properties>
     </profile>
     <profile>
-      <id>demo</id>
```

Comment:
Please keep this profile, there are some test or development scenarios, do not want to compile and package demos, this will consume a lot of time and storage space

Reply:
I don‘t know where to use it.